vcf-to-gds-wf.wdl

```diff
@@ -190,43 +187,39 @@ task check_gds {
 				py_thisChr = py_splitstring[1][0:1]
 			return [py_thisVcfWithSpace, py_thisChr]
 
-		def write_config(py_file):
+		def write_config(py_vcf, py_gds):
 			f = open("check_gds.config", "a")
 
 			# write VCF file
 			f.write("vcf_file ")
-			py_thisVcfSplitOnChr = py_file.split("chr")
-			f.write("'")
-			f.write(vcf_or_gds_with_space(py_thisVcfSplitOnChr)[0])
-			f.write("'")
+			f.write("'" + split_n_space(py_vcf.split("chr"))[0] + "'")
 
 			# write GDS file
 			f.write("\ngds_file ")
-			py_thisGdsSplitOnChr = "~{gds}".split("chr")
-			f.write("'")
-			f.write(vcf_or_gds_with_space(py_thisGdsSplitOnChr)[0])
-			f.write("'")
+			f.write("'" + split_n_space(py_gds.split("chr"))[0] + "'")
+
+			# write the sample file
+			if(~{sample}):
```

Comment:
I don't have experience w/ optional files passed into Python in a task body. Does this test work?

Reply:
Oops! https://github.com/aofarrel/TOPMed_Pipeline_First_Step_WDL/issues/24